gcsio/src/main/java/com/google/cloud/hadoop/gcsio/FileInfo.java

```diff
@@ -132,6 +132,41 @@ public boolean exists() {
     return verificationAttributes == null ? null : verificationAttributes.getMd5hash();
   }
 
+  /** Returns location of the file */
+  public String getLocation() {
```

Comment:
Can we store `itemInfo` instead of extracting each value ?

Reply:
Ok, I will do that